### PR TITLE
Fix Iceberg Glue materialized view test

### PR DESCRIPTION
**Pull request by alexjo2144**
## Description

Test fix, see linked PR for original code change.

> Is this change a fix, improvement, new feature, refactoring, or other?

Fix, test only

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Test only

> How would you describe this change to a non-technical end user or system administrator?

I would not

## Related issues, pull requests, and links

Related code change: https://github.com/trinodb/trino/pull/12522

We forgot to run the Glue suite for it

## Documentation

(x) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

### CODE REVIEW

File: plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogMaterializedViewTest.java [hunk 1]
```diff
@@ -69,6 +71,12 @@ protected String getSchemaName()
         return schemaName;
     }
 
+    @Override
+    protected String getSchemaDirectory()
+    {
+        return new File(schemaDirectory, schemaName) + ".db";
```
Comment (findepi) [inline plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/glue/TestIcebergGlueCatalogMaterializedViewTest.java:77]: move `+ ".db"` inside the brace and add `.toString()`

**Review by findepi — COMMENTED** [review]
is this "Fix" or "Refactor in  preparation to ..." ?

if Fix, please explain what was broken.

**Comment by alexjo2144** [comment]
Updated

**Comment by findepi** [comment]
Test PR with secrets: https://github.com/trinodb/trino/pull/12711
(cc @ilfrin @nineinchnick for potentially streamlining this)

**Comment by alexjo2144** [comment]
Just pushed a fixup, there was another test with the same problem I didn't see at first

**Review by polaris6 — APPROVED** [review]
Thanks!

**Comment by findepi** [comment]
Test PR with secrets green

**Pull request merged**